Sushmitha/c program files/program7.c

```diff
@@ -0,0 +1,38 @@
+// 7. Write a program in C to find the second largest element in an array.
+
+#include <stdio.h>
+void main()
+{
+    int arr[10],i,j=0,large1,large2;
+    printf("Enter 5 elements in the array :\n");
+    for(i=0;i<5;i++)
+    {
+	   scanf("%d",&arr[i]);
+	  }
+   large1=0;
+   for(i=0;i<5;i++)
+   {
+      if(large1<arr[i])
+	     {
+           large1=arr[i];
+           j = i;
+       }
+   }
+  large2=0;
+  for(i=0;i<5;i++)
+  {
+     if(i==j)
+        {
+          i++;
+		      i--;
+        }
+      else
+        {
+          if(large2<arr[i])
+	           {
+               large2=arr[i];
+             }
+        }
+  }
+  printf("The Second largest element is: %d\n", large2);
+}
```

Comment:
![image](https://user-images.githubusercontent.com/30323270/65419718-eab9b380-de1c-11e9-8369-c9b19d7339f6.png)

How second largest is 1 ?